aoa-jupyter-lab

```diff
@@ -147,13 +144,35 @@ RUN echo "${NB_USER} ALL=(ALL) NOPASSWD: /usr/bin/java" >> /etc/sudoers
 #ENV REQUESTS_CA_BUNDLE=/opt/conda/ssl/cert.pem
 
 # Add AOA CLI config file
-RUN mkdir -p /home/jovyan/.aoa && \
-    echo 'aoa_credentials: YWRtaW46YWRtaW4=' >> /home/jovyan/.aoa/config.yaml && \
-    echo 'aoa_url: http://aoa-core:8080' >> /home/jovyan/.aoa/config.yaml   && \
-    echo 'auth_mode: basic' >> /home/jovyan/.aoa/config.yaml
+RUN mkdir -p /home/${NB_USER}/.aoa && \
```

Comment:
i think we can remove this now. It was only really added when we this was our main demo env but it doesn't make sense to have these default credentials for anyone anymore

Reply:
what about the docker-compose stack?

Reply:
even then though, you have to set it up for the given installation? it only made sense when we used the docker-compose stack locally for demos and we didn't want to have to do aoa configure every time we started it